### ISSUE TITLE
论文中训练过程的y =αt * flip(ϵ) − σt *z't ， 这里y为什么要用噪声减去加噪后的图片，而且权重还不一样？diffusion的loss应该只是预测噪声吧，loss =mse(f - flip(ϵ))

### ISSUE THREAD

**Issue by Tinyyuumi** [jeanne-wang/svd_keyframe_interpolation#5]
![image](https://github.com/user-attachments/assets/a0602bb1-4527-455a-9b6c-fcd1dc76d962)

**Comment by jeanne-wang** [comment]
SVD adopts v-prediction, please checkout this part in the original paper for more details.

**Comment by zhou-linpeng** [comment]
![wecom-temp-183bb940cb0c3b475c723e5d1d666147](https://github.com/user-attachments/assets/a3ec686a-cf32-4420-ba85-6d3512706893)
I found that in the training code, **ref_model_pred** was not used in the loss calculation, which seems inconsistent with the description in the paper.
